central/imagev2/datastore/datastore.go

```diff
@@ -25,6 +25,7 @@ type DataStore interface {
 	GetImageMetadata(ctx context.Context, id string) (*storage.ImageV2, bool, error)
 	GetManyImageMetadata(ctx context.Context, ids []string) ([]*storage.ImageV2, error)
 	GetImagesBatch(ctx context.Context, ids []string) ([]*storage.ImageV2, error)
+	GetImageNamesWithDigest(ctx context.Context, digest string) ([]*storage.ImageName, error)
```

Comment:
nit: perhaps `GetImageNames` is enough? - digest is implied from the param.

Reply:
Yeah that should be enough